### ISSUE TITLE
cpu怎么训练

### ISSUE THREAD

**Issue by djhons** [sml2h3/dddd_trainer#8]
我修改了config文件中的GPU为false。

**Comment by huozhenlin** [comment]
兄弟，看看我这是啥错。
<img width="872" alt="image" src="https://user-images.githubusercontent.com/19356940/195619264-109c51f6-6f73-4824-95d0-761bacc2d1de.png">

**Comment by zhx47** [comment]
> 

看一下 _init_.py的176行，如果word为true，那么默认也是使用gpu训练。可以将word改为true，或者修改175 176行代码

**Comment by Sawyerxu** [comment]
.../dddd_trainer/nets/__init__.py的176行修改：
``` python
def get_loss(self, predict, labels, labels_length):
    labels = torch.autograd.Variable(labels)
    if self.word:
        if torch.cuda.is_available(): # 添加判断cuda是否可用
            loss = self.loss(predict, labels.long().cuda())
        else:
            loss = self.loss(predict, labels.long())
    else:
        log_predict = predict.log_softmax(2)
        seq_len = torch.IntTensor([log_predict.shape[0]] * log_predict.shape[1])
        loss = self.loss(log_predict.cpu(), labels, seq_len, labels_length)
    self.optimizer.zero_grad()
    loss.backward()
    self.optimizer.step()

    return loss.item(), self.scheduler.state_dict()['_last_lr'][-1]
